pkg/kubectl/resource/builder.go

```diff
@@ -402,6 +402,7 @@ func (b *Builder) Do() *Result {
 	if b.requireNamespace {
 		helpers = append(helpers, RequireNamespace(b.namespace))
 	}
+	helpers = append(helpers, FilterNamespace())
```

Comment:
I though the point of the resource builder was to allow creating different types of resources at the same time?